docs/pages/component/methods.mdx

```diff
@@ -12,26 +12,44 @@ This page shows the list of available methods
 
 Take the player out of fullscreen mode.
 
-### `pause`
```

Comment:
I don't understand why you move pause, It is the first one in alphabetical order !